modules/n1ql/examples/transactions/results.jsonc

```diff
@@ -60,26 +61,27 @@
         "name": "Keon Hoppe",
         "price": 964.13,
         "route": "63986",
-        "user": "0" // <.>
+        "user": "0" 
       }
     ]
-  },
+  }, 
   {
     "_sequence_num": 7,
     "_sequence_query": "\n\n-- Set a second savepoint\nSAVEPOINT s2;",
     "_sequence_query_status": "success",
     "_sequence_result": {
       "results": []
     }
-  },
-  {
+  }, // end::user0[]
```

Comment:
Not tested this, but I don't think AsciiDoc tagging will work if the tag comment is not on a line by itself